common/scaffolding/reports2/text-level-en.html.html

```diff
@@ -0,0 +1,151 @@
+
+<html>
+<body>
+<nav>
+<b> (a element)</b>
```

Comment:
Use heading to identify section. See related WCAG technique: [G140](https://www.w3.org/WAI/WCAG22/Techniques/general/G140), [G115](https://www.w3.org/WAI/WCAG22/Techniques/general/G115), [G141](https://www.w3.org/WAI/WCAG22/Techniques/general/G141), [F2](https://www.w3.org/WAI/WCAG22/Techniques/failures/F2), [F43](https://www.w3.org/WAI/WCAG22/Techniques/failures/F43)

For example, that line should have been coded like:

```suggestion
<h2><code>a</code> element</h2>
```